### ISSUE TITLE
moveTo function not working

### ISSUE THREAD

**Issue by dynamic75** [alvarotrigo/fullPage.js#1072]
Hi Álvaro, thanks for the great plug in.
I'm having a an issue with the following handler to an an anchor tag outside of the menu used for the rest of the slides (due to layout). But it's not moving the slides. I'm not sure why it doesn't work, no errors show in the console.
```javascript
               $('a[href="#home"]').click(function(){
			$.fn.fullpage.moveTo(1);
			console.log("home link cliked")
		});
```
Here is my mark up:
<!doctype html>

 <html class="no-js" lang=""> 
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title>dynamic75.com Home</title>
        <meta name="description" content="Dynamic75.com is the portfolio of Damian McCleod - web developer">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="apple-touch-icon" href="apple-touch-icon.png">

		<link href='http://fonts.googleapis.com/css?family=Bitter' rel='stylesheet' type='text/css'>
		<link href='http://fonts.googleapis.com/css?family=Roboto+Condensed' rel='stylesheet' type='text/css'>
        <link rel="stylesheet" href="css/jquery.fullPage.css">
		<link rel="stylesheet" href="css/normalize.min.css">
        <link rel="stylesheet" href="css/main.css">
		

        
    </head>
    <body>
        

        <div class="header-container">
            <header class="wrapper clearfix">				
                <h2 class="title">
					<a href="#home">
					<img src="/dynamic75/images/dm-logo.png" alt="Home of dynamic75 - web developer" />
					</a>
				</h2>
                <nav>
                    <ul id="topMenu">
                        <li data-menuanchor="work"><a href="#work">work</a></li>
                        <li data-menuanchor="about"><a href="#about">about</a></li>
                        <li data-menuanchor="contact"><a href="#contact">contact</a></li>
                    </ul>
                </nav>
            </header>
        </div>

        <div class="main-container">
            <div class="main wrapper clearfix">
			
				<div id="fullpage">
					<section class="section" data-anchor="home">
						<div class="splash">					
							<header>
								<div class="trans-box"></div>
								<h1>Hi, my name is Damian. I conceptualize, draw and code. I believe if your basics are solid, the rest is cake (preferably chocolate).</h1>
							</header>                    
						</div>
					</section>
					<section class="section" data-anchor="work"> 	work </section>
					<section class="section" data-anchor="about"> About </section>
					<section class="section" data-anchor="contact"> Contact </section>
				</div>	
				
            </div> 
        </div> 

        <div class="footer-container">
            <footer class="wrapper">
                <ul>
					<li>&copy; copyright 2015 dynamic75</li>
                    <li><a href="#">work</a></li>
                    <li><a href="#">about</a></li>
                    <li><a href="#">contact</a></li>
                </ul>
            </footer>
        </div>
		
		<script src="https://google-code-prettify.googlecode.com/svn/loader/run_prettify.js" autoload="true"></script>
        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.11.2.min.js"><\/script>')</script>

        <script src="js/plugins.js"></script>
        <script src="js/main.js"></script>
		<script src="js/vendor/jquery.fullPage.min.js"></script>

        
        <script>
            (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
            function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
            e=o.createElement(i);r=o.getElementsByTagName(i)[0];
            e.src='//www.google-analytics.com/analytics.js';
            r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
            ga('create','UA-XXXXX-X','auto');ga('send','pageview');
        </script>
    </body>
</html>

**Comment by alvarotrigo** [comment]
You don't need to call the method moveTo if you are already using an anchor link.
The plugin will scroll just with the link.
